src/main/java/edu/ithaca/dragon/wildlife/Trainer.java

```diff
@@ -60,4 +60,33 @@ public String viewParty(){
         }
         return partyInfo;
     }
+
+    public void addToParty(Animal animal){
+        Boolean i = false;
+        int z = 0;
+        if(animals[3] != null){
```

Comment:
This may just be some confusion on my end, but are we doing party sizes of 4? I thought we were doing 6 like pokemon.